### ISSUE TITLE
The command with option --stdout wrongly prints the output to stdout

### ISSUE THREAD

**Issue by Michalski-Piotr** [ailog-analytics/wls-analytics#4]
Hello,

In ```wlsanalytics soa index lolo58 --stdout``` command the new line characters are not printed correctly. Instead of new line there is ``` \n-------------------------------\n ``` in the output:

Example:

```console


[oracle@k8s-registry wlsanalytics]$ wlsanalytics soa index lolo58 --stdout | head -5
b'index_id    : lolo58\ncomposite   : UpdateSalesOrderSiebelCommsProvABCSImpl\nversion     : 2.9.0.0\nlog_file    : /u01/oracle/wlsanalytics/data/o2c/soa_server4/logs/soa_server4.out12242\nindex_file  : /u01/oracle/wlsanalytics/data/index/wlsa-soaerrors-133.index\n\n<Oct 10, 2023 12:00:26,747 AM UTC> <Error> <oracle.soa.bpel.engine.dispatch> <BEA-000000> <Transaction rolledback, Transaction key = Xid=BEA1-2D9D4150B02C08136700(779154420),Status=Rolled back. [Reason=Unknown],numRepliesOwedMe=0,numRepliesOwedOthers=0,seconds since begin=14,seconds left=306,useSecure=false,XAServerResourceInfo[SOADataSource_o2c-domain]=(ServerResourceInfo[SOADataSource_o2c-domain]=(state=rolledback,assigned=soa_server4),xar=SOADataSource,re-Registered = false),XAServerResourceInfo[eis/wls/Queue]=(ServerResourceInfo[eis/wls/Queue]=(state=rolledback,assigned=soa_server4),xar=weblogic.connector.transaction.outbound.XAWrapper@218afc2c,re-Registered = false),XAServerResourceInfo[WLStore_o2c-domain__WLS_soa_server4]=(ServerResourceInfo[WLStore_o2c-domain__WLS_soa_server4]=(state=rolledback,assigned=soa_server4),xar=WLStore_o2c-domain__WLS_soa_server41649769529,re-Registered = false),XAServerResourceInfo[XrefDataSource_o2c-domain]=(ServerResourceInfo[XrefDataSource_o2c-domain]=(state=rolledback,assigned=soa_server4),xar=XrefDataSource,re-Registered = false),SCInfo[o2c-domain+soa_server4]=(state=rolledback),properties=({weblogic.transaction.partitionName=DOMAIN, weblogic.jdbc.affinity.prfmwnexa.PRFMWNEXA.prod.uk=oracle.ucp.jdbc.oracle.OracleConnectionAffinityContext: versionNumber=1.0, isForInstanceAffinity=true, connectionPoolID=null, instanceName=prfmwn1, dbUniqName=prfmwnexa, serviceName=PRFMWNEXA.prod.uk, weblogic.jdbc.affinity.prfmxexa.PRFMX=oracle.ucp.jdbc.oracle.OracleConnectionAffinityContext: versionNumber=1.0, isForInstanceAffinity=true, connectionPoolID=null, instanceName=prfmx1, dbUniqName=prfmxexa, serviceName=PRFMX}),local properties=({weblogic.jdbc.jta.SOADataSource=[ No XAConnection is attached to this TxInfo ], weblogic.jdbc.jta.XrefDataSource=[ No XAConnection is attached to this TxInfo ]}),OwnerTransactionManager=ServerTM[ServerCoordinatorDescriptor=(CoordinatorURL=soa_server4+uk5050yr:8104+o2c-domain+t3+ CoordinatorNonSecureURL=soa_server4+uk5050yr:8104+o2c-domain+t3+ coordinatorSecureURL=null, XAResources={eis/wljms/COMMS_CREATECUST_GENERIC_PRODUCER, eis/aqjms/Topic, eis/XAFileAdapterForHA, eis/wljms/COMMS_PROVISIONINGORDER_CONSUMER, eis/wljms/COMMS_UPDATESALESORDER_GENERIC_PRODUCER, eis/wljms/COMMS_FULFILLMENTORDERUPDATE_PRODUCER, eis/wljms/COMMS_CREATECUST_SBL_CONSUMER, eis/wljms/COMMS_ORDERBILLING_PRODUCER, eis/oracle/in-memory, eis/aqjms/EDNLocalTxDurableTopic, eis/wljms/COMMS_CREATECUST_GENERIC_CONSUMER, eis/wljms/COMMS_UPDATESALESORDER_SBL_PRODUCER, SOADataSource_o2c-domain, eis/wljms/COMMS_CLOUD_SUBMITORDER_CONSUMER, eis/wljms/COMMS_CREATECUST_OSM_CONSUMER, eis/jms/aiaB2BQueueCF, eis/wljms/COMMS_SYNCPRICELIST_GENERIC_CONSUMER, eis/wljms/COMMS_UPDATESALESORDER_GENERIC_CONSUMER, eis/jms/aiaErrorTopicCF, eis/tibjms/Queue, eis/wljms/COMMS_CREATECUST_SBL_PRODUCER, eis/fioranomq/Topic, oracle.tip.adapter.jms.JmsXAResource, eis/wljms/COMMS_SYNCPRODUCT_SBL_CONSUMER, eis/wljms/COMMS_UPDATESALESORDER_SBL_CONSUMER, eis/pramati/Queue, eis/wljms/COMMS_QUERYACCOUNT_OSM_CONSUMER, eis/aq/aiaB2BInfraAQCF, eis/wljms/COMMS_ORDERFALLOUT_GENERIC_PRODUCER, eis/aqjms/EDNLocalTxTopic, eis/tibjmsDirect/Topic, eis/wljms/COMMS_SYNCPRICELIST_SBL_PRODUCER, eis/tibjms/Topic, WSATGatewayRM_soa_server4_o2c-domain, eis/Apps/Apps, eis/wljms/COMMS_UPDASSET_OSM_PRODUCER, eis/wljms/COMMS_SPECIALRATINGLIST_CONSUMER, eis/wls/Queue, eis/wljms/COMMS_SYNCPRODUCT_GENERIC_PRODUCER, OraSDPMDataSource_o2c-domain, eis/tibjmsDirect/Queue, eis/wljms/COMMS_PROVISIONINGORDER_PRODUCER, eis/aqjms/EDNxaDurableTopic, eis/wljms/COMMS_ORDERFALLOUT_CONSUMER, eis/wljms/COMMS_CLOUD_SUBMITORDER_PRODUCER, eis/wljms/COMMS_UPDASSET_OSM_CONSUMER, eis/aqjms/EDNxaTopic, WLStore_o2c-domain_SOAJMSFileStore_auto_4, eis/wljms/COMMS_UPDATESALESORDER_CONSUMER, eis/wljms/COMMS_ORDERFALLOUT_PRODUCER, WLStore_o2c-domain_UMSJMSFileStore_auto_4, eis/wls/EDNLocalTxDurableTopic, eis/sunmq/Queue, eis/wljms/COMMS_SYNCPRICELIST_GENERIC_PRODUCER, eis/wljms/COMMS_SYNCPRICELIST_SBL_CONSUMER, eis/activemq/Queue, eis/wljms/COMMS_QUERYACCOUNT_OSM_PRODUCER, XrefDataSource_o2c-domain, tangosol.coherenceTxCCI, eis/File/XAFileAdapter, eis/webspheremq/Queue, eis/File/XAFileAdapter2, eis/wls/EDNLocalTxTopic, eis/XAFileAdapter3, eis/wls/Topic, tangosol.coherenceTx, WLStore_o2c-domain_BPMJMSFileStore_auto_4, eis/XAFileAdapter, WLStore_o2c-domain_AIADataStore_auto_4, eis/wljms/COMMS_CREATECUST_OSM_PRODUCER, eis/wls/EDNxaDurableTopic, eis/AQ/aqSample, eis/wljms/COMMS_SYNCPRODUCT_SBL_PRODUCER, EDNDataSource_o2c-domain, eis/wljms/COMMS_BILLINGCUSTOMERUPDATE_TOPIC_CONSUMER, eis/wls/EDNxaTopic, eis/wljms/COMMS_SYNCPRODUCT_GENERIC_CONSUMER, eis/NEWCO_CORE_LOG_o2c-domain, eis/wljms/COMMS_SUBMITORDER_CONSUMER, eis/aqjms/Queue, WLStore_o2c-domain__WLS_soa_server4, eis/wljms/COMMS_CREATEASSET_OSM_CONSUMER, eis/wljms/COMMS_SUBMITORDER_PRODUCER, eis/wljms/COMMS_QUERYACCOUNT_GENERIC_PRODUCER, eis/jbossmq/Queue, eis/AQ/PortalEventSyncAQ1, PortalEventSyncAQ1DS_o2c-domain, eis/wljms/COMMS_CREATEASSET_OSM_PRODUCER},NonXAResources={})],CoordinatorURL=soa_server4+uk5050yr:8104+o2c-domain+t3+)\n ** Cikey: 14649438572\n ** ComponentDN: default/UpdateSalesOrderSiebelCommsProvABCSImpl!2.9.0.0*soa_59c8c5b2-376a-4dab-8de5-62d7793d4b4f/UpdateSalesOrderSiebelCommsProvABCSImplProcess\n ** FlowId: 1336892361\n ** Set of Audit Events in currently rolledback transaction: \n-------------------------------\n ** Scope Id: BpSeq15.12\n ** Audit Event Date: Tue, 10 Oct 2023 00:00:12.024 UTC\n ** Audit Message: Completed assign\n ** Audit Detail: null\n ** Audit Event Attributes: \n   ** wikey: 14649438572-BpAss16-BpSeq15.12-2\n   ** label: Transform_UpdateSalesOrderEBMtoSiebelABM\n   ** state: 5\n-------------------------------\n ** Scope Id: BpScp3.13\n ** Audit Event Date: Tue, 10 Oct 2023 00:00:12.024 UTC\n ** Audit Message: Block skipped\n ** Audit Detail: null\n ** Audit Event Attributes: \n   ** wikey: 14649438572-BpDPE1-BpScp3.13-1\n   ** label: Skipping\n   ** state: 5\n-------------------------------\n ** Scope Id: BpSeq15.12\n ** Audit Event Date: Tue, 10 Oct 2023 00:00:12.024 UTC\n ** Audit Message: bpelx:exec executed\n ** Audit Detail: null\n ** Audit Event Attributes: \n   ** wikey: 14649438572-BxExe14-BpSeq15.12-3\n   ** label: TraceLog_BeforeInvokeUpdateUpsert\n   ** state: 5\n-------------------------------\n ** Scope Id: BpSeq18.16\n ** Audit Ev
```
This is not a big problem, but it would be nice to enhance display in the next version.

Regards,
Piotr Michalski

**Comment by tomvit** [comment]
There was a problem with printing out the output which was wrongly using `print` instead of `sys.stdout.write` functions. This was fixed in https://github.com/ailog-analytics/wls-analytics/commit/4fbc1fe081fd74931de7d10f40f7730863bea568.

**Comment by tomvit** [comment]
Fixed in release v0.3.2: https://pypi.org/project/wls-analytics/
